examples/torch/common/execution.py

```diff
@@ -121,4 +121,4 @@ def set_seed(config):
         manual_seed(config.seed)
         cudnn.deterministic = True
         cudnn.benchmark = False
-        torch.set_deterministic(True)
+        torch.use_deterministic_algorithms(True)
```

Comment:
We still claim a support of torch 1.8.2. Shouldn't be `if` condition for different torch version here?

Reply:
They wouldn't deprecate the old way without a new in place. The 1.8.1 [docs](https://pytorch.org/docs/1.8.1/generated/torch.use_deterministic_algorithms.html#torch.use_deterministic_algorithms) describe the new function, so the change keeps our desired compatibility scope.